can_tools/scrapers/official/MD/md_state.py

```diff
@@ -0,0 +1,77 @@
+import pandas as pd
+
+import us
+from can_tools.scrapers import CMU, DatasetBase
+from can_tools.scrapers.official.base import ArcGIS
+
+
+class Maryland(ArcGIS, DatasetBase):
+    """
+    Fetch county level covid data from Maryland's ARCGIS dashboard
+    """
+
+    ARCGIS_ID = "njFNhDsUCentVYJW"
+    has_location = True
+    location_type = "county"
+    state_fips = int(us.states.lookup("Maryland").fips)
+    source = "https://coronavirus.maryland.gov/"
+
+    def fetch(self):
+        return self.get_all_jsons(
+            "MDH_COVID_19_Dashboard_Feature_Layer_Counties_MEMA", 0, ""
+        )
+
+    def normalize(self, data):
+        df = self.arcgis_jsons_to_df(data)
+        df.columns = [x.lower() for x in list(df)]
+        df["location"] = (self.state_fips * 1000) + df["county_fip"].astype(int)
+
+        crename = {
+            "totalcasecount": CMU(
+                category="cases", measurement="cumulative", unit="people"
+            ),
+            "totaldeathcount": CMU(
+                category="deaths", measurement="cumulative", unit="people"
+            ),
+            "total_pop_tested": CMU(
+                category="unspecified_tests_total",
+                measurement="cumulative",
+                unit="unique_people",
+            ),
+            "total_testing_vol": CMU(
+                category="unspecified_tests_total",
+                measurement="cumulative",
+                unit="test_encounters",
+            ),
+            "daily_testing_vol": CMU(
+                category="unspecified_tests_total",
+                measurement="new",
+                unit="test_encounters",
+            ),
+        }
+        out = (
+            df.melt(id_vars=["location"], value_vars=crename.keys())
+            .assign(
+                dt=self._retrieve_dt("US/Eastern"), vintage=self._retrieve_vintage()
+            )
+            .query("location not in (12998, 12999)")
```

Comment:
I don't think this filter shouldn't be necessary -- FIPS code `12XXX` corresponds to counties in Florida.

Reply:
Hello!

Great feedback, thank you! Sorry about the Florida FIPS code - guess you can see what I based this off of :) 

I can definitely create a separate scraper for the state, that shouldn't be too bad.  I'll take a crack at this tomorrow evening and upload when its done.

After looking into it deeper, I agree with CTP that "total_testing_vol" and "daily _testing_vol" are likely a count of specimens and not necessarily test encounters - cross referencing the % positivity rate by county which is [on a daily per-"test" basis](https://coronavirus.maryland.gov/pages/methodology), against the incremental case count which is on a unique-person basis, I see some counties with >40% test overhead for a single day.  Since I doubt that many unique people sought-out two test encounters in such a short span of time as to get both results on the same day, I believe the specimen-basis is more likely.  

![image](https://user-images.githubusercontent.com/75241831/102564198-28e16980-4098-11eb-81d7-6852c543f5d2.png)

Do either of you see a flaw in this analysis?  I think it may be somewhat inflated if the labs are reporting any replicate confirmatory tests for single positive samples, but I would think splitting one specimen into two in the lab is effectively the same as having collected two from the individual.

Reply:
Hey @ChrisBremer I think your analysis here is spot on.

I agree with specimens!